docs/docs/installation.mdx

```diff
@@ -0,0 +1,359 @@
+---
+sidebar_label: Installation
```

Comment:
Are we sure we want this file in? II thought it was removed during the docs update? @m-vdb

Reply:
Not sure, it was a by-product of merge. It exists on main https://github.com/RasaHQ/rasa/blob/main/docs/docs/installation.mdx so put it back in but need someone to confirm 🙏

Reply:
no it was removed, it needs to go away :)